app/controllers/concerns/api/v1/exception_handler.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module ExceptionHandler
+      extend ActiveSupport::Concern
+
+      included do
+        rescue_from ActiveRecord::RecordNotFound do |e|
+          render json: { error: e.message }, status: :not_found
+        end
+        rescue_from ::UnauthorizedException do |e|
+          render json: { error: e.message }, status: :unathorized
+        end
+        rescue_from Exception do |e|
+          render json: { error: e.message }, status: 500
```

Comment:
For this generic exception at least, it would be good to have something like

` render json: { error: Rails.env.production? ? 'Unkown error' : e.message }, status: 500`

So no one can see what happens in our prod environment.